engine/src/main/scala/cromwell/engine/workflow/lifecycle/execution/WorkflowExecutionActor.scala

```diff
@@ -420,7 +420,7 @@ final case class WorkflowExecutionActor(workflowId: WorkflowId,
 
   onTransition {
     case _ -> toState if toState.terminal =>
-      workflowLogger.info(s"$tag done. Shutting down.")
+      workflowLogger.debug(s"$tag done. Shutting down WorkflowExecutionActor.")
```

Comment:
Good call. I'll try to find a better form of words (without the actor name). And since this is now debug (hooray!), I won't worry too much about prettiness